src/sentry/middleware/ratelimit.py

```diff
@@ -92,7 +96,6 @@ def process_view(
                 )
                 if rate_limit_cond:
                     request.will_be_rate_limited = True
-                    enforce_rate_limit = getattr(view_class, "enforce_rate_limit", False)
                     if enforce_rate_limit:
```

Comment:
I think this check can also be removed since `enforce_rate_limit` will always be `True` here